Server/rainbow-server/Sources/Application/Model/EntryPersistence.swift

```diff
@@ -100,18 +100,40 @@ extension ScoreEntry {
         }
         
         static func getLeaderBoardData(from client: CouchDBClient, completion: @escaping (_ entries: [ScoreEntry]?, _ error: Error?) -> Void) {
+            getDatabase(from: client) { database, error in
+                guard let database = database else {
+                    return completion(nil, error)
+                }                                
+                
+                database.queryByView("leader-board", ofDesign: "LeaderBoard", usingParameters: [Database.QueryParameters.descending(true)], callback: { (documents, error) in
+                    guard let documents = documents else {
+                        return completion(nil, error)
+                    }
+                    var entries = [ScoreEntry]()
+                    for document in documents["rows"].arrayValue {
+                        if let newEntry = ScoreEntry(document: document["value"]["doc"], _id: document["id"].stringValue) {
+                            entries.append(newEntry)
```

Comment:
this line is where we can add logic to nullify some of the fields we don't need, such as `objects`, `deviceIdentifier`, and etc.